docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/default.md

```diff
@@ -86,6 +92,8 @@ When prompted, upload the Yugabyte license file that you received from Yugabyte,
 
 If you are prompted to choose an installation type, choose **Online**.
 
+Choose the appropriate sofrware version to install, if you have options to do so.
```

Comment:
Should be changed to:

If you are offered a choice of software versions, select the one that meets your requirements.

Reply:
Agree